lumicks/pylake/detail/utilities.py

```diff
@@ -30,6 +30,39 @@ def first(iterable, condition=lambda x: True):
     return next(x for x in iterable if condition(x))
 
 
+def replace_key_aliases(dictionary, *key_alias_lists):
+    """Replace key aliases by a specific one.
+
+    Parameters
+    ----------
+    dictionary : dict
+        Python dictionary.
+    *key_alias_lists : list
+        Each argument is a list of key aliases. When a key in this list is found, it is converted
+        to the first alias for that key in the list.
+
+    Returns
+    -------
+    dict
+        Dictionary with keys replaced by their first alias.
+
+    Raises
+    ------
+    ValueError
+        If multiple keys are provided for the same alias.
+    """
```

Comment:
alternative or in conjunction with my previous comment, could include an example for extra clarity
```suggestion
    Raises
    ------
    ValueError
        If multiple keys are provided for the same alias.
        
    Example
    --------
    def plot_line(x, y, **kwargs):
        default_kwargs = {"color": "red"}
        plt.plot(x, y, **default_kwargs | replace_key_aliases(kwargs, ["color", "c"])
    """
```

Reply:
I think the example syntax is a little different with the `::`, but good idea, I will add it.